test/testsuite/lantest_dircache_stats.c

```diff
@@ -0,0 +1,375 @@
+/* Standard C library includes */
+#include <errno.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+/* External globals from lantest.c */
+extern bool Debug;
+
+/* Constants */
+#define LOG_BUFFER_SIZE (100 * 1024)  /* 100KB buffer for reading log file */
+#define MAX_LOG_LINES_DISPLAY 10      /* Number of log lines to show when no stats found */
+#define CONFIG_LINE_SIZE 1024         /* Max line size for config file parsing */
+
+/* Debug logging macro */
+#define DEBUG_LOG(fmt, ...) \
+    do { \
+        if (Debug) \
+            fprintf(stderr, "DEBUG: " fmt "\n", ##__VA_ARGS__); \
+    } while(0)
+
+#ifdef __linux__
+
+/* Parse netatalk config file to extract log file path */
+static char *parse_config_for_log_path(void)
+{
+    FILE *config_fp = NULL;
+    char *log_file_path = NULL;
+    const char *config_paths[] = {
+        "/etc/afp.conf",
+        "/etc/netatalk/afp.conf",
+        "/usr/local/etc/afp.conf",
+        "/usr/local/etc/netatalk/afp.conf",
+        NULL
+    };
```

Comment:
You shouldn't have to look for afp.conf – the C program can get the accurate location with `_PATH_CONFDIR "afp.conf"`.

For instance, the afpldaptest application does this:

https://github.com/Netatalk/netatalk/blob/1d115d0cc9159d389fbbd70d5093706892e7b6f9/bin/misc/uuidtest.c#L57

Then extracts the information it needs from the iniconfig dictionary.

Reply:
Great. Again wasn't sure where this was available